app/models/mhv_account.rb

```diff
@@ -17,6 +17,14 @@ class MhvAccount < ActiveRecord::Base
   ALL_STATES = %i(unknown needs_terms_acceptance ineligible registered upgraded register_failed upgrade_failed).freeze
   # Range of VHA facility IDs at which people receive treatment
   PATIENT_FACILITY_RANGE = [358, 758].freeze
+  ADDRESS_ATTRS = %w(street city state postal_code country).freeze
+  UNKNOWN_ADDRESS = {
+    address1: 'Unknown Address',
+    city: 'Washington',
+    state: 'DC',
+    zip: '20571',
```

Comment:
NITPICK: So 20001 is the more popular D.C. zip code, not sure I have seen 20571 before.